log function causes panic on cr deletion

https://github.com/openstack-k8s-operators/openstack-ansibleee-operator/blob/181d81e78f04fbf21111209a8588c68079564e72/controllers/openstack_ansibleee_controller.go#L238


this log can panic 
```
2023-03-06T13:15:43-05:00	DPANIC	controllers.OpenStackAnsibleEE	odd number of arguments passed as key-value pairs for logging	{"ignored key": {"metadata":{"creationTimestamp":null},"spec":{},"status":{}}}
github.com/openstack-k8s-operators/openstack-ansibleee-operator/controllers.(*OpenStackAnsibleEEReconciler).getOpenStackAnsibleeeInstance
	/home/centos/repos/ansibleee-operator/controllers/openstack_ansibleee_controller.go:238
github.com/openstack-k8s-operators/openstack-ansibleee-operator/controllers.(*OpenStackAnsibleEEReconciler).Reconcile
	/home/centos/repos/ansibleee-operator/controllers/openstack_ansibleee_controller.go:73
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:122
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:323
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:274
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:235
2023-03-06T13:15:43-05:00	INFO	Observed a panic in reconciler: odd number of arguments passed as key-value pairs for logging	{"controller": "openstackansibleee", "controllerGroup": "ansibleee.openstack.org", "controllerKind": "OpenStackAnsibleEE", "OpenStackAnsibleEE": {"name":"nova-compute-1-deploy-nova","namespace":"openstack"}, "namespace": "openstack", "name": "nova-compute-1-deploy-nova", "reconcileID": "cfc542c8-48e1-42bd-8d76-9d2059fa4766"}
panic: odd number of arguments passed as key-value pairs for logging [recovered]
	panic: odd number of arguments passed as key-value pairs for logging

goroutine 469 [running]:
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile.func1()
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:119 +0x31c
panic({0x209fc60, 0xc00079a310})
	/usr/lib/golang/src/runtime/panic.go:890 +0x267
go.uber.org/zap/zapcore.CheckWriteAction.OnWrite(0x2, 0xc0008d2750, {0x0, 0xc0007aa600, 0x577})
	/home/centos/go/pkg/mod/go.uber.org/zap@v1.24.0/zapcore/entry.go:198 +0x9b
go.uber.org/zap/zapcore.(*CheckedEntry).Write(0xc0008d2750, {0xc0004799c0, 0x1, 0x1})
	/home/centos/go/pkg/mod/go.uber.org/zap@v1.24.0/zapcore/entry.go:264 +0x624
go.uber.org/zap.(*Logger).DPanic(0xc000222460, {0x23e54f0, 0x3d}, {0xc0004799c0, 0x1, 0x1})
	/home/centos/go/pkg/mod/go.uber.org/zap@v1.24.0/logger.go:248 +0x6c
github.com/go-logr/zapr.(*zapLogger).handleFields(0xc0004faed0, 0x0, {0xc00079a2d0, 0x1, 0x1}, {0x0, 0x0, 0x0})
	/home/centos/go/pkg/mod/github.com/go-logr/zapr@v1.2.3/zapr.go:147 +0xc89
github.com/go-logr/zapr.(*zapLogger).Info(0xc0004faed0, 0x0, {0x23f3e9d, 0x4c}, {0xc00079a2d0, 0x1, 0x1})
	/home/centos/go/pkg/mod/github.com/go-logr/zapr@v1.2.3/zapr.go:210 +0xd2
github.com/go-logr/logr.Logger.Info({{0x2535f08, 0xc0004faed0}, 0x0}, {0x23f3e9d, 0x4c}, {0xc00079a2d0, 0x1, 0x1})
	/home/centos/go/pkg/mod/github.com/go-logr/logr@v1.2.3/logr.go:261 +0x12b
github.com/openstack-k8s-operators/openstack-ansibleee-operator/controllers.(*OpenStackAnsibleEEReconciler).getOpenStackAnsibleeeInstance(0xc0008edf00, {0x2533b38, 0xc000856990}, {{{0xc00086e750, 0x9}, {0xc00004a6a0, 0x1a}}})
	/home/centos/repos/ansibleee-operator/controllers/openstack_ansibleee_controller.go:238 +0x24a
github.com/openstack-k8s-operators/openstack-ansibleee-operator/controllers.(*OpenStackAnsibleEEReconciler).Reconcile(0xc0008edf00, {0x2533b38, 0xc000856990}, {{{0xc00086e750, 0x9}, {0xc00004a6a0, 0x1a}}})
	/home/centos/repos/ansibleee-operator/controllers/openstack_ansibleee_controller.go:73 +0xcd
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Reconcile(0xc0003a0140, {0x2533b38, 0xc000856990}, {{{0xc00086e750, 0x9}, {0xc00004a6a0, 0x1a}}})
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:122 +0x172
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).reconcileHandler(0xc0003a0140, {0x2533b38, 0xc000856990}, {0x21d9f00, 0xc00005aac0})
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:323 +0x3f7
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).processNextWorkItem(0xc0003a0140, {0x2533a90, 0xc0008ecec0})
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:274 +0x3a5
sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2.2()
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:235 +0xdc
created by sigs.k8s.io/controller-runtime/pkg/internal/controller.(*Controller).Start.func2
	/home/centos/go/pkg/mod/sigs.k8s.io/controller-runtime@v0.14.4/pkg/internal/controller/controller.go:231 +0x4d9
```

--- a/controllers/openstack_ansibleee_controller.go
+++ b/controllers/openstack_ansibleee_controller.go
@@ -235,11 +235,11 @@ func (r *OpenStackAnsibleEEReconciler) getOpenStackAnsibleeeInstance(ctx context
 			// Request object not found, could have been deleted after reconcile request.
 			// Owned objects are automatically garbage collected. For additional cleanup logic use finalizers.
 			// Return and don't requeue
-			r.Log.Info("OpenStackAnsibleEE resource not found. Ignoring since object must be deleted", instance)
+			r.Log.Info("OpenStackAnsibleEE resource not found. Ignoring since object must be deleted")
 			return &redhatcomv1alpha1.OpenStackAnsibleEE{}, nil
 		}
 		// Error reading the object - requeue the request.
-		r.Log.Error(err, err.Error(), instance)
+		r.Log.Error(err, err.Error())
 		return instance, err
 	}
 

